polaris-migrator/src/migrations/replace-sass-color/color-maps.ts

```diff
@@ -0,0 +1,219 @@
+import type {Exact} from 'type-fest';
+
+export interface ColorHue {
+  [hue: string]: ColorValue;
+}
+
+export interface ColorValue {
+  [value: string]: string;
+}
+
+const createMap = <T extends Exact<ColorHue, T>>(config: T): T => config;
```

Comment:
**thought:** I think this can be removed. Unless you need to narrow the values to literal string types this constrained identity function shouldn't be need.